nservicebus/msmq/index.md

```diff
@@ -51,6 +51,9 @@ The DISM command line equivalent is:
 DISM.exe /Online /NoRestart /English /Enable-Feature /FeatureName:MSMQ-Container /FeatureName:MSMQ-Server
 ```
 
+## MSMQ Machine name limitation
+
+For MSMQ to function properly, [the server name should be 15 characters or less](http://geekswithblogs.net/Plumbersmate/archive/2012/02/03/make-sure-computer-names-are-15-characters-or-less-fro.aspx). This is because of [NETBIOS limitation](https://support.microsoft.com/en-us/help/163409/netbios-suffixes-16th-character-of-the-netbios-name). Having a longer machine name may result in MSMQ not functioning properly. 
```

Comment:
Since I don't have any authoritative MS link that connects the two, I am not being forceful here.